rtgui/profilestore.h

```diff
@@ -208,11 +208,16 @@ class ProfileStore :
         return internalDynamicEntry;
     }
 
-    std::vector<DynamicProfileRule> &getDynamicProfileRules()
+    const std::vector<DynamicProfileRule> &getDynamicProfileRules() const
     {
         return dynamicRules;
     }
 
+    void setDynamicProfileRules(const std::vector<DynamicProfileRule> &r)
```

Comment:
> though the former would be faster with an r-value-ref (which isn't how it's used here).

Since I'm the outsider here, I can go that way if you prefer... but if you
want to hear my opinion (and that's an if :-), this is totally immaterial in this context. I picked the const version for consistency with other parts of the code

> if you turned std::vector<DynamicProfileRule> dynamicRules into std::unique_ptr<std::vector<DynamicProfileRule>> dynamicRules. There are already so many unnecessary header dependencies in RT, we shouldn't carelessly add more

Sure, I'll do that! :+1: